appveyor.yml

```diff
@@ -12,15 +12,38 @@ environment:
   CYGMIRROR: http://ftp.inf.tu-dresden.de/software/windows/cygwin32
   CYGCACHE: C:\cygwin64\var\cache\setup
   opam_url: https://github.com/fdopen/opam-repository-mingw/releases/download/0.0.0.1/opam64.tar.xz
+  DESTCOQ: C:\coq64_inst
+  matrix:
+  - USEBUILDSCRIPT: false
+  - USEBUILDSCRIPT: true
```

Comment:
`USEBUILDSCRIPT: false` is for running the test-suite with OPAM
`USEBUILDSCRIPT: true` is for building the Coq packages
Given how convoluted this is, short comments giving this info in the appveyor file wouldn't hurt (e.g. on the two lines above).